src/modules/flow/netctl/netctl.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "netctl",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "netctl",
+      "description": "Send list of all network services attached to board.",
+      "methods": {
+        "open": "open_list_services",
+	"close": "close_list_services"
+      },
+      "name": "netctl/list-services",
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "Name of network services supported by system.",
+          "name": "OUT"
```

Comment:
Can you give me one related example for ADDED or REMOVED port . I cannot find how to use ADDED or REMOVED doc in soletta. And all the services would get from service_callback in sol_netctl_add_service_monitor,  calling sol_netctl_get_services maybe result in that all the services are not ready yet. For example, if there are 5 services in the network environment, monitor will  asyn get service to upper layer one by one, so calling sol_netctl_get_service would get wrong result.  Because I am not familar with ADDED/REMOVED port, these port can resolve this problem?